packages/network/src/v2/schemas/decodeData.ts

```diff
@@ -0,0 +1,59 @@
+import { getStaticByteLength } from "@latticexyz/schema-type";
+import { hexToArray } from "@latticexyz/utils";
+import { TableSchema } from "../common";
+import { decodeStaticField } from "./decodeStaticField";
+import { decodeDynamicField } from "./decodeDynamicField";
+
+export const decodeData = (schema: TableSchema, hexData: string): Record<number, any> => {
```

Comment:
ooc when do you use `const` and when `function` for functions?

Reply:
oof, its inconsistent!

I generally prefer `const` but I've started using `export function` a bit more lately because you can overload type signatures with it, and you can't do that for `const` definitions.

Reply:
I think the `function` definitions are a bit more common in the current codebase, we should align everything at some point for consistency. Not blocking for this PR though.